### ISSUE TITLE
Templates for arguments

### ISSUE THREAD

**Issue by tats-u** [jean-humann/docs-to-pdf#197]
```
--contentSelector="article" --paginationSelector="a.pagination-nav__link.pagination-nav__link--next" --excludeSelectors=".margin-vert--xl a,[class^='tocCollapsible'],.breadcrumbs,.theme-edit-this-page"
```

This software always requires a so-long options. It is so long that no one can input without reading the README. It would be nice if we can shorten this to like:

```
--template docusaurus2
```

**Comment by jean-humann** [comment]
Hi @tats-u , Thanks for the participation 👍 
I agree with you on this point.
I was wondering about a subcommand such as 
`docs-to-pdf docusaurus` encapsulating all the default flag link to docusaurus. Coud link to a `--from-build` flag as well.
Will do the same for other documentation framework later.
What do you think about it ?

**Comment by tats-u** [comment]
Did you mean:

- default: general purposes
- subcommands: optimized for specific documentation frameworks e.g. Docusaurus v1 / v2

It's acceptable. Could you tell me the details of `--from-build`?

**Comment by jean-humann** [comment]
@tats-u exactly
`--from-build` would allow you to build PDF directly from the the build directory generated by `docusaurus build` without having to do `docusaurus serve`

**Comment by tats-u** [comment]
@jean-humann It looks great.
`--version` or similar flag will be necessary to designate the version of Docusaurus.

**Comment by jean-humann** [comment]
@tats-u Done in #212 with `--version` flag
